src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/ApplicationsTransformer.kt

```diff
@@ -144,6 +144,7 @@ class ApplicationsTransformer(
         status = getStatusFromSummary(domain),
         type = "CAS1",
         tier = domain.getTier(),
+        isWithdrawn = domain.getIsWithdrawn(),
```

Comment:
It might be worth tweaking the commit title to be "Add '**is**Withdrawn property..." for added clarity and in case people are search the history with the property name in the future?